src/cleanTest.py

```diff
@@ -270,33 +270,121 @@ def makeLists(list): # Make 2 lists from list of list containing 2 items
 
 # In[9]: User input
 
-user_input = input("Please enter file path: ")
+user_input = input("Please enter file name: ")
 
-# THIS NEEDS TO BE WORKED ON STILL - WILL BE UPDATED NOV 30
-# def classifier(user_input, data):
-  # articles = getText(data)
-  # articles = cleanData(articles)
+def classifier(user_input):
+  with open(DATA_DIRECTORY / user_input, encoding = "utf8") as csvfile:
+    user_data = list(csv.reader(csvfile))
   
-  # full_data = getText(data)
-
-  # fake_dict, true_dict = makeDictionary(full_data, label_list)
+  user_articles = getText(user_data)
+  user_articles = cleanData(user_articles)
```

Comment:
Because we first get the text column from the file and then we run it through the clean function - I can change the name for one of them but they're both the articles